llama-index-integrations/llms/llama-index-llms-mistral-rs/README.md

```diff
@@ -0,0 +1,3 @@
+# LlamaIndex Llms Integration: Mistral-Rs
+
+Please install the Python bindings according to the guide [here](https://github.com/EricLBuehler/mistral.rs/blob/master/mistralrs-pyo3/README.md).
```

Comment:
Could we add a "Usage" section here. This README gets displayed in our llamahub.ai so its good to put useful content here for potential users of this integration.